docs/contributor_guide/contributing_example_notebooks.rst

```diff
@@ -1,16 +1,16 @@
 Contributing example notebooks
 ------------------------------
 
-To contribute new example notebooks please submit a pull request. We have
-certain requirements that need to be satisfied to contribute a notebook. Those
-include:
+We'd love to collaborate with anyone interested in describing scenarios for using Fairlearn!
 
-* Clear structure
-* Appropriate framing of the problem while respecting that fairness is a
-  fundamentally sociotechnical challenge
-* Usage of at least some part of Fairlearn
+When discussing and critiquing examples, here are some lenses to consider:
 
-Good examples of existing notebooks that abide by these requirements are:
+Microrubric for critiques:
+"""""""""""""""""""""""""""""""""""
+1. **Deployment context**: Describes a real deployment context, not just a dataset.
+2. **For developers**: Speaks the language of developers and data scientists.  Considers real practitioner needs.  Fits within the lifecycle of real practioner work.  See `Holstein et al (2019) <https://arxiv.org/pdf/1812.05239.pdf>`_, `Madaio et al. (2020) <http://www.jennwv.com/papers/checklists.pdf>`_.
+3. **Sociotechnical**: Models the team's value that fairness as a sociotechnical challenge.    Avoids abstraction traps.  See `Selbst et al. (2020) <https://andrewselbst.files.wordpress.com/2019/10/selbst-et-al-fairness-and-abstraction-in-sociotechnical-systems.pdf>`_.
+4. **Open**:  Discusses tradeoffs and compares alternatives.  Illustrates how Fairlearn contributes.
```

Comment:
This one is not entirely clear to me. With 'open', do you mean that it should be clear that the approach in the notebook is *a* way to approach it, rather than *the* way? Or do you mean that trade-offs should be made explicit? Or perhaps both or something else?
And do you specifically mean to illustrate how Fairlearn contributes to *making trade-offs*? I feel 'show how Fairlearn contributes' might even be a separate point.

Reply:
Yeah, all those kinds of things! 👍 

The idea is to talk about "is this teaching people decisions and rationale and thought behind what is happening in the notebook?"  So maybe this is a better frame:

> **Not just code**:  Discusses tradeoffs and compares alternatives.  Describes why using Fairlearn makes sense.

Does that capture what you're thinking of?

Reply:
I think most of the other points also cover that it's not just code, I would probably call it something like 'substantiated process' (or 'workflow', 'trade-offs'). This is the closest I get to translating the word that I have in mind in Dutch, haha. Nitpicking: Fairlearn as a project is supposed to be more than just code, so perhaps we should explicitly reference the software/functionalities/features/... in this rubric? 

So maybe something like: 

> **Substantiated**: Discusses trade-offs and compares alternatives. Describes why using particular Fairlearn functionalities makes sense.

Please pardon my non-native English ;)

Reply:
👍 Sounds good to me!  "justified" could be another similar word, but this all sounds like the right universe of things to me :)